examples/amp-story/ads/remote.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ad-attributes": {
+      "type": "doubleclick",
+      "data-slot": "/30497360/a4a/story_360"
+    }
+}
```

Comment:
nit: newline you should be able to configure your editor to automatically include these.

Reply:
Yes, found out how to do so [https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files](Here)